configure.ac

```diff
@@ -653,15 +635,8 @@ AC_SUBST(SEDOBJ)
 
 if test "$prefix" = "NONE"; then
 	AC_DEFINE_UNQUOTED(IRCD_PREFIX, "$ac_default_prefix", [Prefix where the ircd is installed.])
-
 else
-
-dnl Don't get bitten by Cygwin's stupidity if the user specified
-dnl a custom prefix with a trailing slash
-
-	prefix=`echo $prefix | sed 's/\/$//'`
```

Comment:
Let's keep this line, in case someone is crazy enough to use `/` as prefix.  POSIX allows for leading `//` to be treated specially, and I have personally worked in an environment where that is the case (not that I'd run an ircd in that environment, but people do crazy things).

Reply:
Wow, you're right, I've missed that part of POSIX.   I guess it's for network shares or something? I just readded that line with a better comment.  Thanks!